de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/session/internal/SarosSession.java

```diff
@@ -957,52 +974,71 @@ public void removeActivityConsumer(IActivityConsumer consumer) {
 
     @Override
     public boolean isShared(IResource resource) {
-        return projectMapper.isShared(resource);
+        return referencePointMapper.isShared(resource);
     }
 
     @Override
     public List<IResource> getSharedResources() {
-        return projectMapper.getPartiallySharedResources();
+
+        return referencePointMapper.getPartiallySharedResources();
     }
 
     @Override
-    public String getProjectID(IProject project) {
-        return projectMapper.getID(project);
+    public String getReferencePointID(IReferencePoint referencePoint) {
+        return referencePointMapper.getID(referencePoint);
     }
 
     @Override
-    public IProject getProject(String projectID) {
-        return projectMapper.getProject(projectID);
+    public IReferencePoint getReferencePoint(String referencePointID) {
+        return referencePointMapper.getReferencePoint(referencePointID);
     }
 
+    /*
+     * Map<IReferencePoint, List<IResource>> referencePointResourceMap =
+     * projectMapper .getReferencePointResourceMapping(); Map<IProject,
+     * List<IResource>> projectResourceMap = new HashMap<IProject,
+     * List<IResource>>();
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This block of commented-out lines of code should be removed. ([squid:CommentedOutCodeLine](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3ACommentedOutCodeLine))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=210&issues=AWVWjH5glRPS_ea-bThb&open=AWVWjH5glRPS_ea-bThb)

[ISSUE_KEY]: AWVWjH5glRPS_ea-bThb